src/caffe/layers/cudnn_deconv_layer.cpp

```diff
@@ -247,7 +254,7 @@ void CuDNNDeconvolutionLayer<Dtype>::Reshape(
         break;
       }
     }
-    if(!found_conv_algorithm) LOG(ERROR) << "cuDNN did not return a suitable algorithm for deconvolution.";
+    if(!found_conv_algorithm) CUDNN_CHECK(cudnnStatus_t::CUDNN_STATUS_BAD_PARAM);
```

Comment:
Same here.

Reply:
fml, I won't make a new release for a whitespace fix.

also this is a caffe fork, where is the coding style doc?